dvc/command/add.py

```diff
@@ -69,6 +70,9 @@ def add_parser(subparsers, parent_parser):
         help="Specify name of the DVC-file this command will generate.",
         metavar="<filename>",
     )
+    parser.add_argument(
+        "--desc", type=str, metavar="<description>", help="Data description.",
```

Comment:
I think we use pseudo var types in `metavar`? Like
```suggestion
        "--desc", type=str, metavar="<string>", help="Data description.",
```
Would just need to double check with existing ones (maybe use `<text>` instead if it's already out there).

Reply:
Also, `Data description`: Maybe we can be slightly more informative like "User description of the data (optional)". Maybe even append "This has no effect for DVC." or something like that (if that's true)?